core/client.py

```
@@ -116,56 +117,140 @@ def __init__(self, config: AstrBotConfig, plugin_data_dir: Path):
+                "server_address",
+                "port",
+                "auth_key",
+                "takeout_id",
+                "tmp_auth_key",
+            ]
+            if TelegramClientWrapper._telethon_reads_takeout_id_as_tmp_key():
+                columns.remove("tmp_auth_key")
+                takeout_idx = columns.index("takeout_id")
+                columns.insert(takeout_idx, "tmp_auth_key")
+            return columns
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)